packages/lib/src/toolbar/Toolbar.module.css

```diff
@@ -4,7 +4,7 @@
   z-index: 1; /* for toolbar menus to appear above visualizations (overflow, selectors) */
   display: flex;
   min-width: 0;
-  height: 2.5rem;
+  height: var(--h5w-toolbar--height, 2.5rem);
```

Comment:
As mentioned, the default values are smaller to avoid visual regressions in existing applications.